bundles/org.openhab.binding.somfytahoma/README.md

```diff
@@ -60,59 +60,63 @@ Please see the example below.
+| on/off heating system                                                         |   target_heating_level  | target heating level (off, eco, comfort, frostprotection)                                                                   |
+| heating system                                                                |   current_temperature   | current temperature of the heating system                                                                                   |
+| heating system                                                                |      current_state      | current state of the heating system                                                                                         |
+| heating system, thermostat                                                    |    target_temperature   | target temperature of the heating system                                                                                    |
+| heating system, thermostat                                                    |      battery_level      | battery level of the heating system                                                                                         |
+| thermostat                                                                    |       heating_mode      | standard heating mode of the thermostat (away, freeze, manual, ...)                                                         |
+| thermostat                                                                    | derogation_heating_mode | derogation heating mode of the thermostat (away, freeze, manual, ...)                                                       |
+| thermostat                                                                    |  derogation_activation  | derogation activation state (inactive, active)                                                                              |
+| temperature sensor                                                            |       temperature       | temperature reported by the sensor                                                                                          |
+| myfox camera, myfox alarm                                                     |       cloud_status      | cloud connection status                                                                                                     |
+| myfox camera                                                                  |          shutter        | controlling of the camera shutter                                                                                           |
+| myfox alarm                                                                   |   myfox_alarm_command   | used for sending commands to Somfy Myfox alarm device                                                                       |
+
+
+REMARKS:
```

Comment:
Subsubheader? Prefixed by three hashtags and CamelCase?

```suggestion
### Remarks:
```

Reply:
OK